packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -187,13 +440,27 @@ class GrpcGatewayRouteSpec extends GatewayRouteSpec {
    */
   readonly routeTarget: IVirtualService;
 
+  /**
+   * The default target host name to write to.
+   * Boolean is used to select either enable or disable.
+   * When enabled, rewrites the original request received at the Virtual Gateway to the destination Virtual Service hostname.
+   * When disabled, does not rewrite to destination Virtual Service hostname.
+   *
+   * @default true
+   */
+  readonly defaultHostname?: boolean;
```

Comment:
Same here, let's call this `rewriteRequestHostname` instead.

Reply:
Addressed